### DISCUSSION TITLE
Single Sign-On (SSO) Support

### DISCUSSION THREAD

**Discussion by Sudarsh1010** [keizerworks/keizer-auth-api#30] [Ideas]
- **Description**: Provide integrations for SSO with popular identity providers (e.g., Google, Microsoft, GitHub) for a streamlined login experience.
- **Why**: Many users expect SSO as a feature, and it simplifies user management for clients who already rely on external identity providers.

**Comment by aayushchugh** [comment]
We should definitely build this feature but for now, let's complete MVP first. We can put this in v1.1.0 milestone

**Reply by Sudarsh1010** [reply]
Yeah, thought the same, It will also take some time for me to get deeper understanding of this feature. I will be keep on adding resources here.
